File: app/src/main/java/com/lexicalanalyzer/nusret/analyzer/Utils/Node.java
package com.lexicalanalyzer.nusret.analyzer.Utils;

/**
 * Created by Halil on 11.03.2018.
 */

public class Node {

    Word node_word;
    Node left;
    Node right;

    public Node(Word node_word) {
        this.node_word = node_word;
        left = null;
        right = null;
    }

}

File: app/src/main/java/com/lexicalanalyzer/nusret/analyzer/Utils/Node.kt
package com.lexicalanalyzer.nusret.analyzer.Utils

/**
 * Created by Halil on 11.03.2018.
 */

class Node(internal var node_word: Word) {
    internal var left: Node? = null
    internal var right: Node? = null

    init {
        left = null
        right = null
    }

}
